build.gradle

```diff
@@ -28,6 +28,14 @@ ext.ext_classpath = ['com.google.code.gson:gson:2.2.4',
   'io.dropwizard.metrics:metrics-core:3.1.2']
 ext.target_dir = "$distsDir"
 ext.target_java8_dir = "${ext.target_dir}/java8"
+ext.target_javadoc_dir = "${ext.target_dir}/docs/javadoc"
+
+if (JavaVersion.current().isJava8Compatible()) {
+  tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Sorry, I don't know why the behavior might be different / why it was working in the ant build.

Unrelated to this particular problem but something to keep in mind: eventually we'll want to address QUARKS-152.  Don't know if your current work / changes will help.